docs/contributor-info/releasing.md

```diff
@@ -3,7 +3,7 @@
 We're now releasing this as a combined ruby gem plus npm package. We will keep the version numbers in sync.
 
 ## Testing the Gem before Release from a Rails App
-See [Contributing](../../CONTRIBUTING.md)
+See [Contributing](https://github.com/shakacode/react_on_rails/tree/master/CONTRIBUTING.md)
```

Comment:
@Judahmeek we might want this as part of the regular docs published to shakacode.com

Reply:
I agree. I think we also might want to move Projects & Kudos into /testimonials